$NetBSD: patch-ai,v 1.2 2014/08/21 13:21:33 jperkin Exp $

gmime2.4 fix

Place xdg files in pkgsrc standard location.
Not to be submitted upstream.

--- jbsrc/jb.c.orig	2008-05-22 15:47:04.000000000 +0000
+++ jbsrc/jb.c
@@ -166,7 +166,7 @@ jb_package_configure (void)
   jb_require_packages("GNOME", "gnome", "glib-2.0 >= 2.14 gthread-2.0 gconf-2.0 >= 2.4.0 gtk+-2.0 >= 2.12 libgnomeui-2.0 >= 2.14.0 gnome-vfs-2.0 libglade-2.0 libxml-2.0 libnotify >= 0.4.1");
   jb_require_packages("D-Bus", "dbus", "dbus-glib-1");
 
-  jb_check_packages_for_options("GMime", "gmime", "gmime-2.0 >= 2.2.7",
+  jb_check_packages_for_options("GMime", "gmime", "gmime-2.4",
 				"hotmail",
 				"imap",
 				"maildir",
@@ -376,7 +376,7 @@ jb_package_add_resources (void)
   group = jb_group_new("data");
 
   jb_group_add_desktop_file(group, "mail-notification-properties.desktop.in", "$datadir/applications");
-  jb_group_add_desktop_file(group, "mail-notification.desktop.in", "$sysconfdir/xdg/autostart");
+  jb_group_add_desktop_file(group, "mail-notification.desktop.in", "$datadir/applications");
 
   jb_group_add_resource(group, JB_GROUP_RESOURCE(jb_template_new("mail-notification.schemas.in.in")));
   jb_group_add_gconf_schemas(group, "mail-notification.schemas.in");
@@ -433,7 +433,7 @@ jb_package_add_resources (void)
 					"DATADIR", "$datadir",
 					"PKGDATADIR", "$pkgdatadir",
 					"LIBDIR", "$libdir",
-					"GNOMELOCALEDIR", "$datadir/locale",
+					"GNOMELOCALEDIR", "$localedir",
 					NULL);
 
   /*
